src/app/large-display/large-display.component.ts

```diff
@@ -8,13 +8,21 @@ import { TimerBarComponent } from 'src/app/timer-bar/timer-bar.component'
 })
 export class LargeDisplayComponent implements OnInit {
 
+  componentHeight: number;
+
   @ViewChild('timerBar', {static: true}) timerBar: TimerBarComponent;
-  constructor() {   
-    
+
+  constructor() { }
+
+  // resize the component height with the window
+  @HostListener('window:resize', ['$event']) onResize(event) {
+    this.componentHeight = window.innerHeight-83;
```

Comment:
I feel like you could do the resizing with CSS instead of this.

Reply:
Setting it to 100% was the first thing I tried when I started working on it yesterday. Changing the height in the css would be preferable, but it doesn't work the way you would expect.